spec/enumerables_spec.rb

```diff
@@ -0,0 +1,174 @@
+# frozen_string_literal: true
+
+require './lib/enumerable.rb'
+
+describe Enumerable do
+  let(:arr) { [1, 2, 7, 8, 5] }
+
+  describe '#my_each should return' do
+    it 'an array holding all elements of the given array' do
+      expect(arr.my_each).to eql([1, 2, 7, 8, 5])
```

Comment:
**Tip: 💭** 😄 
This is fine for your custom method, however, if you look at the original method you'd notice that when a block is given it returns an enumerator hence a test like following would fail: 
`expect(arr.my_each).to eql(arreach)`